07-routing-gossip.md

```diff
@@ -894,30 +889,10 @@ generated directly by the node itself, so they should ignore filters.
 If a node requires an initial sync of gossip messages, it will be flagged
```

Comment:
Shouldn't we remove the whole `Initial Sync` section? It doesn't really make sense anymore, at least in its current state.